ui/app/AppLayouts/Communities/panels/PermissionsSettingsPanel.qml

```diff
@@ -46,6 +49,40 @@ StackView {
         root.push(newPermissionView, properties, StackView.Immediate);
     }
 
+    SortFilterProxyModel {
+        id: allChannelsTransformed
+
+        sourceModel: root.channelsModel
+
+        proxyRoles: [
```

Comment:
All of these proxy roles could be just a simple `JoinRole`, instead of the expensive `ExpressionRole`s

Reply:
cc @micieslak since you merged the other PR here already :)

Reply:
Good point, can be improved by a chance